survey_contact_generation/views/survey_question_views.xml

```diff
@@ -10,6 +10,23 @@
                     <field name="allowed_field_ids" invisible="1" />
                 </group>
             </xpath>
+            <xpath expr="//field[@name='matrix_row_ids']" position="attributes">
+                <attribute
+                    name="context"
+                >{'default_question_id': active_id, 'default_res_partner_field': res_partner_field}</attribute>
+            </xpath>
+            <xpath
+                expr="//field[@name='matrix_row_ids']//field[@name='value']"
```

Comment:
And logically this is wrong as well. I detected it trying to configure this feature.
```suggestion
                expr="//field[@name='suggested_answer_ids']//field[@name='value']"
```